### DISCUSSION TITLE
2011/06/26/%E6%99%AE%E9%99%80%E5%B1%B1%E6%B8%B8%E8%AE%B0

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#334] [Announcements]
# 2011/06/26/%E6%99%AE%E9%99%80%E5%B1%B1%E6%B8%B8%E8%AE%B0

周末公司组织去普陀山旅游。周五晚上从吴淞码头出发，第二天一早驶达普陀山。我们很不幸，因为周六正赶上台风袭击，整天都在下雨，而我们只能穿着雨衣打着伞游玩了。

https://blog.qizhen.xyz/2011/06/26/%E6%99%AE%E9%99%80%E5%B1%B1%E6%B8%B8%E8%AE%B0

**Comment by ruanqizhen** [comment]
之前博客下的留言：

新酒店网
2011年07月8日 14:39 编辑
海边那张好哦.

回复
汉庭连锁酒店
2011年07月2日 14:59 编辑
不错，不错．

回复
